LevelEditor/.idea/.name

```diff
@@ -0,0 +1 @@
+LevelEditor
```

Comment:
Please remove the .idea directory from the repo, and add a line to the .gitignore file at the root of the repo that says this:

```
.idea/
```